packages/form-footer/src/FormFooter.types.ts

```diff
@@ -83,16 +71,4 @@ export interface FormFooterProps extends HTMLElementProps<'footer'> {
    * Useful for setting left and right margins, or max-width
    */
   contentClassName?: string;
-
-  /**
-   * Styling prop
-   */
-  className?: string;
```

Comment:
do we no longer need this documentation? it seems like it could be helpful to differentiate from `contentClassName`. The current description isn't helpful but maybe could be used to specify which element it applies to?

Reply:
I don't think so. I'll update the documentation of `contentClassName` to specify that this is added to the first `div` child of `footer`